packages/agent-builder/src/processors/write-file.ts

```diff
@@ -0,0 +1,17 @@
+import { writeFile } from 'fs/promises';
```

Comment:
we should remove this, this was just for debugging context windows when we were first iterating